pkg/querier/queryrange/queryrangebase/retry.go

```diff
@@ -57,6 +59,20 @@ func (r retry) Do(ctx context.Context, req Request) (Response, error) {
 	defer func() { r.metrics.retriesCount.Observe(float64(tries)) }()
 
 	var lastErr error
+
+	// For the default of 5 tries
+	// try 0: no delay
+	// try 1: 250ms wait
+	// try 2: 500ms wait
+	// try 3: 1s wait
+	// try 4: 2s wait
+
+	cfg := backoff.Config{
+		MinBackoff: 250 * time.Millisecond,
+		MaxBackoff: 5 * time.Second,
+		MaxRetries: 0,
+	}
+	bk := backoff.New(ctx, cfg)
```

Comment:
Can you add a test please?

Reply:
I don't think I understand what we would test here?

Reply:
Was initially thinking it'd validate the delays you'd defined there, but that's tested by the underlying lib. Disregard